docs/Translation.md

```diff
@@ -83,6 +83,7 @@ You can find translation packages for the following languages:
 - Finnish (`fi`): [aikain/ra-language-finnish](https://github.com/aikain/ra-language-finnish)
 - French (`fr`): [marmelab/ra-language-french](https://github.com/marmelab/react-admin/tree/master/packages/ra-language-french)
 - German (`de`): [greenbananaCH/ra-language-german](https://github.com/greenbananaCH/ra-language-german)
+- German (`de`) messages for [ra-tree-ui-materialui](https://github.com/marmelab/react-admin/tree/master/packages/ra-tree-ui-materialui) package: [straurob/ra-tree-language-german](https://github.com/straurob/ra-tree-language-german)
```

Comment:
can you put that on the same line as the main German translation, and under a shorter form, like `([tree translation](xxx))`